libs/framework/src/service_registry.c

```diff
@@ -1215,4 +1215,9 @@ static void celix_waitForPendingRegisteredEvents(celix_service_registry_t *regis
         count = (long)hashMap_get(registry->pendingRegisterEvents.map, (void*)svcId);
     }
     celixThreadMutex_unlock(&registry->pendingRegisterEvents.mutex);
+}
+
+long celix_serviceRegistry_nextSvcId(celix_service_registry_t* registry) {
+    long scvId = __atomic_fetch_add(&registry->nextServiceId, 1, __ATOMIC_SEQ_CST);
```

Comment:
If I am right, this return 0. "These built-in functions perform the operation suggested by the name, and return the value that had previously been in *ptr".
So now instead of starting at 1, we start at 0? I don't think this introduces some error, just curious.

Reply:
Correct. svc id 0 is a valid service id, but celix always started with 1.

I will revert this. Even though svc id 0 is valid, this change can trigger some dormant issues where validity of service id are tested as svcId > 0. 
That is also why the test where updated.